js/donations-visualiser.js

```diff
@@ -190,6 +190,9 @@ var data_request = d3.json("data/all_data.json")
         $("#loading-modal").modal('hide');
         processData(data);
 
+        // zoom out a ways initially
+        zoomTo(0.5).event(container);
```

Comment:
Up to you :) I found the initial 1:1 value a bit intense with the current force config.. lots of nodes zooming across the viewport rather than within it.. don't think this is a real solution, but my taste is for a more stable intro..